### PR TITLE
created function to count binary gap

### PR DESCRIPTION
**Pull request by gabrielplacinta1**

### CODE REVIEW

File: task1/task1.cs [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+// using System.Collections.Generic;
+
+// you can write to stdout for debugging purposes, e.g.
+// Console.WriteLine("this is a debug message");
+
+class Solution {
+    public int solution(int N) {
+        var bin = Convert.ToString(N, 2);
+        var maxLength = 0;
+        var counter = 0;
```
Comment (GeorgeVragalev) [inline task1/task1.cs:12]: you can optimize the code by checking, if the bin has less than 2 1's then automatically the gap is 0

**Review by GeorgeVragalev — CHANGES_REQUESTED** [review]
try to do it in 1 loop

File: task1/task1.cs (hunk 1 above)
Comment (denisburlacu) [reply task1/task1.cs:12]: instead of using ToString, try to iterate per bits

Comment (denisburlacu) [reply task1/task1.cs:12]: using bitwise operator

**Pull request merged**